src/createAnimatedComponent.tsx

```diff
@@ -179,14 +191,18 @@ function inlinePropsHasChanged(styles1: StyleProps, styles2: StyleProps) {
   return false;
 }
 
-function getInlinePropsUpdate(inlineProps: Record<string, any>) {
+function getInlinePropsUpdate(
+  inlineProps: TransformsStyle['transform'] | Record<string, any>
```

Comment:
I suspect that `TransformsStyle['transform']` satisfies `Record<string, any>`, what's the point of having an union type here?

Reply:
TransformStyle is an array, so it doesn't